Directory.Build.targets

```diff
@@ -78,6 +78,8 @@
   <ItemGroup Condition="'$(Stage)' == 'dev' AND '$(OutputType)' != 'Exe' AND '$(Api)' != 'false'">
     <AssemblyAttribute Include="System.Diagnostics.CodeAnalysis.ExperimentalAttribute">
       <_Parameter1>TBD</_Parameter1>
+      <_Parameter2>UrlFormat = "https://aka.ms/dotnet-extensions-warnings/{0}"</_Parameter2>
+      <_Parameter2_IsLiteral>true</_Parameter2_IsLiteral>
     </AssemblyAttribute>
   </ItemGroup>
```

Comment:
@geeknoid FYI this essentially will be a no-op as the compiler won't provide support the assembly level - as noted in https://github.com/dotnet/roslyn/pull/68702.

Reply:
@terrajobst Do you have a recommendation on how to deal with this?

Right now, in this repo you can indicate a package is in dev stage. This means the whole package is in development, every API it contains is inherently experimental and the package may in fact be removed. With the Experimental attribute not being enforced at the assembly level, it would require engineers to mark every single public type in the assembly as experimental in order to make sure customers don't take an unintended dependency on the API.

Is this right? Shouldn't experimental support assembly scope?

Reply:
@terrajobst Ping

Reply:
This was discussed [here](https://github.com/dotnet/runtime/pull/85444#discussion_r1235718243).

Basically, this doesn't work for obsolete today and we concluded we'd rather want to align with that for now and revisit if this is identified as a limitation. When I spec'ed it, I did include module and assembly level because that's what we did for the preview feature and platform features before and it seemed convenient, pretty much for the reason you're spelling out.

I think I'd prefer if the feature allowed for assembly level, but we should probably enable this for obsoletion as well then for consistency.

@jaredpar @jcouv what do you guys think? Should I file a separate issue so we can discuss this? No idea what the cost of such a change would be.

Reply:
Yes, please file an issue. We (compiler) generally prefer module-level over assembly-level, but I understand that multi-module scenario is not a scenario with Core... So that might need some discussion.

Reply:
From discussion with Aleksey, supporting both assembly- and module-level would work and make sense for this attribute, without causing complications.